functional.py

```diff
@@ -0,0 +1,62 @@
+
+def sequential_map_2(*args):
+    *functions_list, coll = [*args]
```

Comment:
`[*args]` - эта конструкция избыточна
```suggestion
    *functions_list, coll = args
```